feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1026 @@
+		// Ensure ISIS Adjacency is up on LAG_2
+		if ok := awaitAdjacency(t, dut, aggIDs[1]); !ok {
+			t.Fatal("ISIS Adjacency is Down on LAG_2")
+		}
+		forwardingViableDisable(t, dut, 2, 3)
+		time.Sleep(time.Minute)
+		// Ensure ISIS Adjacency is Down on LAG_2
+		if ok := awaitAdjacency(t, dut, aggIDs[1]); ok {
+			t.Fatal("ISIS Adjacency is Established on LAG_2")
+		}
+		otgutils.WaitForARP(t, ate.OTG(), top, "IPv4")
+		startTraffic(t, dut, ate, top)
+		confirmNonViableForwardingTraffic(t, dut, ate, atePortList[1:6], dutPortList[1:6])
+		// Ensure that traffic from ATE port1 to pfx4 are discarded on DUT
+		verifyTrafficFlow(t, ate, flows[1:2])
+		// Ensure there is traffic received on DUT LAG_3
+		if got := validateLagTraffic(t, dut, ate, dutPortList); got == false {
+			t.Fatal("Packets are not Received on LAG_3")
+		}
+		verifyTrafficFlow(t, ate, flows[0:1])
+	})
+
+	t.Run("Running Testcase RT-5.7.2.3", func(t *testing.T) {
+		t.Logf("Setting Forwarding-Viable to True on the Member Port Port6 of LAG2")
+		forwardingViableEnable(t, dut, 6)
+		startTraffic(t, dut, ate, top)
+		checkBidirectionalTraffic(t, dut, dutPortList[5:8])
+		confirmNonViableForwardingTraffic(t, dut, ate, atePortList[1:5], dutPortList[1:5])
+		verifyTrafficFlow(t, ate, flows)
+	})
+}
+
+// configureDUT configures DUT
+func configureDUT(t *testing.T, dut *ondatra.DUTDevice) []string {
+
+	t.Helper()
+	fptest.ConfigureDefaultNetworkInstance(t, dut)
+
+	configureDUTLoopback(t, dut)
+
+	var aggIDs []string
+	for _, a := range []*aggPortData{agg1, agg2, agg3} {
+		d := gnmi.OC()
+		aggID := netutil.NextAggregateInterface(t, dut)
+		aggIDs = append(aggIDs, aggID)
+		portList := initializePort(t, dut, a)
+		if deviations.AggregateAtomicUpdate(dut) {
+			clearAggregate(t, dut, aggID, a, portList)
+			setupAggregateAtomically(t, dut, aggID, a, portList)
+		}
+		lacp := &oc.Lacp_Interface{Name: ygot.String(aggID)}
+		lacp.LacpMode = oc.Lacp_LacpActivityType_ACTIVE
+		lacpPath := d.Lacp().Interface(aggID)
+		fptest.LogQuery(t, "LACP", lacpPath.Config(), lacp)
+		gnmi.Replace(t, dut, lacpPath.Config(), lacp)
+
+		aggInt := &oc.Interface{Name: ygot.String(aggID)}
+		configAggregateDUT(dut, aggInt, a)
+
+		aggPath := d.Interface(aggID)
+		fptest.LogQuery(t, aggID, aggPath.Config(), aggInt)
+		gnmi.Replace(t, dut, aggPath.Config(), aggInt)
+		if deviations.ExplicitInterfaceInDefaultVRF(dut) {
+			fptest.AssignToNetworkInstance(t, dut, aggID, deviations.DefaultNetworkInstance(dut), 0)
+		}
+		for _, port := range portList {
+			i := &oc.Interface{Name: ygot.String(port.Name())}
+			i.Type = ethernetCsmacd
+			e := i.GetOrCreateEthernet()
+			e.AggregateId = ygot.String(aggID)
+			if deviations.InterfaceEnabled(dut) {
+				i.Enabled = ygot.Bool(true)
+			}
+			if port.PMD() == ondatra.PMD100GBASEFR {
+				e.AutoNegotiate = ygot.Bool(false)
+				e.DuplexMode = oc.Ethernet_DuplexMode_FULL
+				e.PortSpeed = oc.IfEthernet_ETHERNET_SPEED_SPEED_100GB
+			}
```

Comment:
I[ see this being used in many places in featureprofiles tests](https://github.com/search?q=repo%3Aopenconfig%2Ffeatureprofiles+%22Disable+FEC+for+100G-FR+ports+because+Novus+does+not+support+it%22&type=code).  Novus is the hardware ATE we are using and according to this comment, it doesn't support autonegotiation.

I am not aware of using deviations for interop with ATE/OTG equipment, only with the DUT.  My thinking is to allow this.  If we want to make a change to how this need is accommodated in the code, a separate issue can be raised.

Reply:
But that's different. The code you linked refers to the OTG configuration, the code I highlighted is used on the DUT side (and also sets different values)

Reply:
Opened #2977 to investigate

Reply:
Done Added the deviation as well for DUT